### PR TITLE
Java update

### PR DESCRIPTION
**Pull request by jacksolirius**
### Change description ###
Java update for Adoption repo 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-  id 'org.owasp.dependencycheck' version '8.0.2'
+  id 'org.owasp.dependencycheck' version '8.3.1'
   id 'scala'
-  id 'com.github.lkishalmi.gatling' version '3.0.2'
-  //id 'java-gradle-plugin'
-  id 'groovy'
-  id 'com.gradle.plugin-publish' version '0.9.8'
-  id 'maven-publish'
+  id 'java-library'
+  id 'io.gatling.gradle' version '3.9.3.1'
```
Comment (jonsoloway-hmcts) [inline build.gradle:5]: This should be 3.9.5 please

File: build.gradle [hunk 2]
```diff
@@ -26,51 +22,30 @@ tasks.withType(JavaExec) {
...
-  scalaVersion '2.12.12'
-  simulations = [
-          'simulations.adoptionSimulation'
-  ]
-}
-
-test {
-  testLogging {
-    showStandardStreams = true
+  gatlingVersion '3.9.3'
```
Comment (jonsoloway-hmcts) [inline build.gradle:34]: 3.9.5

File: src/gatling/conf/gatling.conf [hunk 3]
```diff
@@ -24,22 +25,53 @@ gatling {
...
       css {
         #cacheMaxCapacity = 200 # Cache size for the compiled CSS selectors queries,  set to 0 to disable caching
       }
     }
-
     directory {
-      #data = user-files/data               # Folder where user's data (e.g. files used by Feeders) is located
-      #bodies = user-files/bodies           # Folder where bodies are located
-      simulations = src/gatling/simulations/simulations # Folder where the bundle's simulations are located
+      #simulations = user-files/simulations # Directory where simulation classes are located (for bundle packaging only)
```
Comment (jonsoloway-hmcts) [inline src/gatling/conf/gatling.conf:34]: Please replace this file with the version for 3.9.5 from the template-performance repo

**Review by jonsoloway-hmcts — COMMENTED** [review]
Comments left

File: build.gradle (hunk 2 above)
Comment (jacksolirius) [reply build.gradle]: Changed

File: src/gatling/conf/gatling.conf (hunk 3 above)
Comment (jacksolirius) [reply src/gatling/conf/gatling.conf]: Changed

**Review by jonsoloway-hmcts — APPROVED** [review]

**Pull request merged**